workspaces/jupyterlab-base/jupyter_server_config.py

```
@@ -21,7 +21,7 @@
 # RTC appears to be broken in the current version of `jupyter-collaboration`:
 # https://github.com/jupyterlab/jupyter-collaboration/issues/162
 # This opt-in will only be useful once the underlying issue is resolved.
-enable_rtc = bool(os.environ.get("ENABLE_REAL_TIME_COLLABORATION", False))
+enable_rtc = bool(os.environ.get("ENABLE_REAL_TIME_COLLABORATION", False))  # ruff:ignore[invalid-envvar-default]
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: invalid-envvar-default

Enable `lint.preview` to use rule names

(RUF102)